plugins/reporters/cyclonedx/src/funTest/kotlin/CycloneDxReporterFunTest.kt

```diff
@@ -30,34 +30,46 @@ import io.kotest.matchers.should
 import io.kotest.matchers.shouldBe
 import io.kotest.matchers.shouldNotBe
 
+import java.io.File
+
 import org.cyclonedx.parsers.JsonParser
 import org.cyclonedx.parsers.XmlParser
 
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.model.config.PluginConfiguration
 import org.ossreviewtoolkit.reporter.ORT_RESULT
 import org.ossreviewtoolkit.reporter.ORT_RESULT_WITH_VULNERABILITIES
 import org.ossreviewtoolkit.reporter.ReporterInput
+import org.ossreviewtoolkit.utils.common.Options
 import org.ossreviewtoolkit.utils.common.normalizeLineBreaks
 import org.ossreviewtoolkit.utils.test.getAssetAsString
 
 class CycloneDxReporterFunTest : WordSpec({
     val defaultSchemaVersion = CycloneDxReporter.DEFAULT_SCHEMA_VERSION.versionString
     val outputDir = tempdir()
 
+    fun CycloneDxReporter.generateReport(result: OrtResult, options: Options): List<File> =
+        generateReport(
+            ReporterInput(result),
+            outputDir,
+            PluginConfiguration(options = options)
```

Comment:
Nit: As the argument is called like the parameter, I believe this could just be `PluginConfiguration(options)` without sacrificing readablity.